apps/desktop/src-electron/app.ts

```diff
@@ -542,11 +542,11 @@ if (!singleInstance && !process.mas) {
   });
 
   app.name = APP_NAME;
-  app.on('ready', () => {
+  app.on('ready', async () => {
     if (!mainWindow) {
       mainWindow = createMainWindow();
     }
-    init();
+    await init();
```

Comment:
```suggestion
    void init();
```

子进程的 init 不用等待执行结果，影响 main window 的展示时机